lib/rules/declaration-block-trailing-semicolon/__tests__/index.js

```diff
@@ -145,3 +145,35 @@ testRule(rule, {
     }
   ]
 });
+
```

Comment:
I don't think there's need for a new `testRule`. Instead, add `fix: true` to the two existing rejects:

- https://github.com/YozhikM/stylelint/blob/5e73c24a05a35985feeef8422aed31efa03c14e8/lib/rules/declaration-block-trailing-semicolon/__tests__/index.js#L41-L92
- https://github.com/YozhikM/stylelint/blob/5e73c24a05a35985feeef8422aed31efa03c14e8/lib/rules/declaration-block-trailing-semicolon/__tests__/index.js#L117-L147